pyston/order/managers.py

```diff
@@ -25,6 +25,9 @@ class ModelOrderManager:
     and fields.
     """
 
+    def _get_real_field_name(self, resource, field_name):
```

Comment:
maybe make it a function? it's the same method multiple times and doesn't use self

Reply:
this is only recalling resource method, This solution adds posibility to rewrite it